rfc/rfc014-authorization-credential.md

```diff
@@ -101,37 +102,39 @@ When `implied`, values MUST be added to the `restrictions` array.
 
 The `id` field MUST contain the DID of the actor.
 The `subject` field MAY contain the patient identifier. In the example above the *oid* for the Dutch citizenship number is used.
-The `service` field refers to a Bolt. A Bolt MUST describe the set of FHIR resources that can be accessed with the credential.
+The `purposeOfUse` field refers to a Bolt. A Bolt MUST describe the set of FHIR resources that can be accessed with the credential.
 The `restrictions` array MAY further limit access.
 When no `subject` is given, the credential MUST contain `restrictions` that refer to individual resources.
 The contents of those individual resources MUST NOT contain any personal information.
 
 ### 3.2.3 Legal base
 
 The patient consent can be either implied or explicit. 
-When it's implied, this should be reflected by the type of service that is entered.
+When it's implied, this should be reflected by the `purposeOfUse`.
```

Comment:
SHOULD or MUST?

Reply:
the MUST is implicated the other way around, from the `purposeOfUse`